app/src/main/res/raw/help.txt

```diff
@@ -3,8 +3,8 @@
 <h3>Web Site: https://zephyrsoft.org/trackworktime</h3>
 <br>
 This app can track your work time easily! You can automate time tracking using geo-fencing functions (see below).
-You may also <b>categorize each recorded interval</b> by a predefined client/task and a free text.
-Of course, the list of clients/tasks can be edited to suit your needs, and the app has a widget for your home screen.<br>
+You may also <b>categorize each recorded interval</b> by a predefined client/task and an arbirary text.
```

Comment:
I'm thankful that you propose better words (I'm not a native english speaker), but please make sure they are spelled correctly.

Reply:
Whoops, sorry for the mistake. I fixed it.